src/slash/open.ts

```diff
@@ -110,36 +110,28 @@ export function formatFriendCode(friendCode: number): string {
 	return `${friendString.slice(0, 3)}-${friendString.slice(3, 6)}-${friendString.slice(6, 9)}`;
 }
 
-async function setNickname(member: GuildMember, friendCode: number): Promise<void> {
-	// check for redundancy
-	const formatCode = formatFriendCode(friendCode);
-	const baseName = member.nickname || member.user.username;
-	if (baseName.includes(formatCode)) {
-		return;
+async function setNickname(member: GuildMember, friendCode: number, ign?: string): Promise<void> {
+	const name = ign || member.user.username;
```

Comment:
Honestly glad to be able to skip the redundancy check now that we're just setting the whole name. Could have done it this way the entire time with just the username...

Reply:
Theoretically it could preserving whatever nickname the user set for themselves while amending, but this isn't a use case.

Reply:
Well we could always go `ign || nickname || username` but yeah it's really not a big deal.